src/components/ActivitiesList/ActivitiesList.js

```diff
@@ -0,0 +1,26 @@
+import Oops from '../Oops/Oops'
+import FlipCard from '../flipCard/FlipCard'
+
+export default function ActivitiesList({ activities, onClickBookmark }) {
+  if (!activities.length) {
+    return <Oops />
+  } else {
+    return activities.map(activity => (
+      <FlipCard
+        id={activity.id}
+        name={activity.name}
+        description={activity.description}
+        street={activity.street}
+        city={activity.city}
+        zipCode={activity.zipCode}
+        country={activity.country}
+        openingHours={activity.openingHours}
+        website={activity.website}
+        isFreeOfCharge={activity.isFreeOfCharge}
+        isBookmarked={activity.isBookmarked}
+        key={activity.id}
+        onClickBookmark={onClickBookmark}
+      />
+    ))
+  }
```

Comment:
I usually prefer doing an early exit instead of an `if...else`:
```suggestion
  if (!activities.length) {
    return <Oops />
  } 
  
  return activities.map(activity => (
      <FlipCard
        id={activity.id}
        name={activity.name}
        description={activity.description}
        street={activity.street}
        city={activity.city}
        zipCode={activity.zipCode}
        country={activity.country}
        openingHours={activity.openingHours}
        website={activity.website}
        isFreeOfCharge={activity.isFreeOfCharge}
        isBookmarked={activity.isBookmarked}
        key={activity.id}
        onClickBookmark={onClickBookmark}
      />
    ))
```

Reply:
👍